static/app/views/replays/replayTable.tsx

```diff
@@ -85,23 +95,38 @@ function ReplayTable({isFetching, replays, showProjectColumn, sort, fetchError}:
   const organization = useOrganization();
   const theme = useTheme();
   const minWidthIsSmall = useMedia(`(min-width: ${theme.breakpoints.small})`);
+  const hasTxEvent = Boolean(replays?.some(replay => replay.txEvent));
```

Comment:
This would be better as a prop passed in: `showTxnColumn` or something like that. As we do with `showProjectColumn`.

Reply:
Updated this the downside that I see though is this prop could be set to true but the table isn't provided a data set with the transaction data. In that case there will be an empty column